### PR TITLE
added enableAzurev1 to cnsconfig

### PR DESCRIPTION
**Pull request by sawyerawesome**
## Description

Added the new `enableAzurev1` variable to cnsconfig
## Motivation and Context
This is allow us to skip queries for tenants with the variable set to false

## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "e2251f5c7ce368e2051c369abaf0b26f2b450ebc",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "71a79dea8881f75233c43202c865265af237876d",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "e2251f5c7ce368e2051c369abaf0b26f2b450ebc",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "520fe2e1ef083b6c82bfbafa7968f4e1d8fd843d",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "e2251f5c7ce368e2051c369abaf0b26f2b450ebc",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "22a58ef342ea92f7b44a1d8664262709d26634f4",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "e2251f5c7ce368e2051c369abaf0b26f2b450ebc",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "3e569e1fceadab96359e3fe0af55e0d2e59211f8",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "e2251f5c7ce368e2051c369abaf0b26f2b450ebc",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "4130e8ee2255357be86d5880d121f36b90747278",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "a373f2df07ae58cf99907ce0be77f21b3afe1f8e",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "4130e8ee2255357be86d5880d121f36b90747278",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "4130e8ee2255357be86d5880d121f36b90747278",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "3c50112d1e67bf94b8edfc636ddbcd8a2fe73d51",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "3c50112d1e67bf94b8edfc636ddbcd8a2fe73d51",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "4130e8ee2255357be86d5880d121f36b90747278",
    "pipeline": "master"
  }
]
```

**Review by ericrpowers — DISMISSED** [review]
I am not sure of our standard with having v1, v2, v3... in an attribute (Should it be `enableAzurev1` or `enableAzureV1`). Considering this is a temporary measure anyway, LGTM

File: specs/cnsconfig.spec [hunk 1]
```diff
@@ -31,6 +31,12 @@ indexes:
 # Attributes
 attributes:
   v1:
+  - name: enableAzurev1
```
Comment (primalmotion) [inline specs/cnsconfig.spec:34]: `azureV1Enabled`. Booleans in api must be in the passive form

**Review by primalmotion — CHANGES_REQUESTED** [review]

File: specs/cnsconfig.spec (hunk 1 above)
Comment (primalmotion) [reply specs/cnsconfig.spec:34]: I realize there are other enableX on this api. if we can still change them, now is the time

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "3c50112d1e67bf94b8edfc636ddbcd8a2fe73d51",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "495e5f7f1c4900eb0d7ddea64921664934307fd3",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "4b16f64b5d2f6ef0bc5d70f953c6e20968d52003",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "34a40befc747cec7d2f2f350f591f79d073d1a1a",
    "pipeline": "master"
  }
]
```

File: specs/cnsconfig.spec (hunk 1 above)
Comment (sawyerawesome) [reply specs/cnsconfig.spec]: https://redlock.atlassian.net/browse/CNS-4883

**Review by ericrpowers — APPROVED** [review]

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "4b16f64b5d2f6ef0bc5d70f953c6e20968d52003",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "e33dc664a752588df066037871c027cdba562fd2",
    "pipeline": "master"
  }
]
```

**Comment by sawyerawesome** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "",
    "component": "backend",
    "pr-id": "1631",
    "commit-sha": "4ed887c1cc98c6f6645f14fa11572346fd771972",
    "pipeline": "master"
  },
  {
    "project": "",
    "component": "gaia",
    "pr-id": "788",
    "commit-sha": "4b16f64b5d2f6ef0bc5d70f953c6e20968d52003",
    "pipeline": "master"
  }
]
```

**Pull request merged**